### ISSUE TITLE
Support for OpenShift objects

### ISSUE THREAD

**Issue by smontri** [kontena/lens#4565]
**What would you like to be added**:
I would like being able to see in Lens UI OpenShift specific objects such as Routes, DeploymentConfig, DeploymentBuild, ...

**Why is this needed**:
Customers running OpenShift clusters are asking to have this view in Lens.
This could maybe come from a Lens Extension, maybe to reactivate the prototype started some time ago.

- Kubernetes distribution: OpenShift
- Desktop OS: macOS

**Comment by jim-docker** [comment]
Yes, this seems good for an extension. Would anyone in the OpenShift community like to take this on?

**Comment by smontri** [comment]
Anyone from the community or Lens team who will take that one?

**Comment by YevheniiPokhvalii** [comment]
That would be really convenient, because we use several Kubernetes clusters and one of them is OpenShift. And we have to switch between Lens and the OpenShift console or CLI to manage oc objects like routes, etc.

**Comment by jorikseldeslachts** [comment]
Would be great to have this.

**Comment by Amarlanda** [comment]
Is there any movement on this, chaps?
